Packages/vcs/vcs/VTKPlots.py

```diff
@@ -1553,3 +1553,9 @@ def png_dimensions(self, path):
         img = reader.GetOutput()
         size = img.GetDimensions()
         return size[0], size[1]
+
+    def canvasraised(self):
```

Comment:
@chaosphere2112 what is this? The name almost suggest this is a query I would expect it to return True/False shouldn't this be called `raiseCanvas` ?